tests/phpunit/tests/Checker/Default_Check_Repository_Tests.php

```diff
@@ -28,14 +28,20 @@ public function test_register_static_check() {
 		$check = new Static_Check();
 		$this->repository->register_check( 'static_check', $check );
 
-		$this->assertSame( array( 'static_check' => $check ), $this->repository->get_checks()->to_map() );
+		$checks_map = $this->repository->get_checks()->to_map();
+
+		$this->assertArrayHasKey( 'static_check', $checks_map );
+		$this->assertInstanceOf( Static_Check::class, $checks_map['static_check'] );
```

Comment:
See my above comment. If we broke `Default_Check_Repository` into separate `Empty_Check_Repository` and `Default_Check_Repository extends Empty_Check_Repository`, we could keep the code in here as is, using `Empty_Check_Repository` instead. That would still cover all logic, in a simpler and safer way.

This test class could then be renamed to `Empty_Check_Repository_Tests`.